.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: node_js
+node_js:
+  - 8
+script:
+  - node_modules/bs-platform/bin/bsb -make-world
```

Comment:
It looks like builds can run faster by adding:
```yaml
cache:
  directories:
    - "node_modules"
```
Since it runs `npm install` automatically on git clone then it should keep it up to date.

Also just thinking, `npm install bs-platform` need to be explicitely run since @bobzhang wanted bs-`platform` to be a peer dependency instead of a full dependency.